sites/svelte.dev/src/routes/docs/+layout.svelte

```diff
@@ -13,6 +16,9 @@
 	</div>
 
 	<div class="page content">
+		{#if category}
+			<p class="category" aria-label="category">{category}</p>
+		{/if}
```

Comment:
If we end up doing this, we should remove the `aria-label` - it doesn't have any effect on `<p>` elements

Reply:
yeah... i'm not good at a11y, i was trying to give some semantics, but did not really know what tag to choose.

Do you think the `<p>` alone is good enough, or is there another tag more relevant ?

Reply:
Or, i just realised there is the `aria-description` that could fit.